great_expectations/profiler/domain_builder/simple_semantic_type_domain_builder.py

```diff
@@ -18,9 +20,11 @@ class SemanticDomainTypes(Enum):
     def _get_domains(
         self,
         *,
-        validator,
-        batch_ids,
-        include_batch_id,
+        validator: Optional[Validator] = None,
+        batch_ids: Optional[List[str]] = None,
+        include_batch_id: Optional[bool] = False,
+        # TODO: <Alex>ALEX -- The signature of this method is inconsistent with that in the base class.</Alex>
```

Comment:
@anthonyburdi I am leaving all comments "ALEX" in the code for now so that at some point (e.g., once the concept document is finalized) we can go over them and make all the useful corrections/modifications.  Thank you.